test/e2e/dedicated_admin_test.go

```diff
@@ -0,0 +1,70 @@
+//+build e2e
+
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	_ "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var _ = Describe("Openshift on Azure dedicated-cluster-admin e2e tests [DedicatedAdmin]", func() {
+	defer GinkgoRecover()
+
+	It("should read nodes", func() {
+		_, err := dadmin.kc.CoreV1().Nodes().Get("master-000000", metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("should have full access on all non-infrastructure namespaces", func() {
+		// Create project as normal user
+		namespace := nameGen.generate("e2e-test-")
+		c.createProject(namespace)
+
+		wait.PollImmediate(2*time.Second, 5*time.Minute, func() (bool, error) {
+			rb, err := dadmin.kc.RbacV1().RoleBindings(c.namespace).Get("admin", metav1.GetOptions{})
+			if err != nil {
+				return false, nil
+			}
+			for _, subject := range rb.Subjects {
+				if subject.Kind == "Group" && subject.Name == "dedicated-admins" {
+					return true, nil
+				}
+			}
+			return false, nil
+		})
+		// get namespace created by user
+		_, err := dadmin.pc.Projects().Get(c.namespace, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		// attempt to delete namespace
+		err = dadmin.pc.Projects().Delete(c.namespace, &metav1.DeleteOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		c.cleanupProject(10 * time.Minute)
```

Comment:
Since you do a project delete as part of the test above, you don't need this.

Reply:
Makes sense.  Removed.